### PR TITLE
#362 violation root user

### LINKED ISSUE #362: Violation if the root user have a active secret key xor no active mfa
There should be a violation, if the root user (`<root_account>`) have a active secret key (first violation) and MFA is not active (second violation)

Right configuration:

| user | mfa_active | access_key_1_active | access_key_2_active |
| --- | --- | --- | --- |
| `<root_account>` | **true** | **false** | **false** |

### PR DESCRIPTION
**Pull request by mrandi**
fixes #362

### CODE REVIEW

File: fullstop-apis/fullstop-violation-api/src/main/java/org/zalando/stups/fullstop/violation/ViolationType.java [hunk 1]
```diff
@@ -54,4 +54,7 @@
     public static final String MISSING_APPLICATION_VERSION_IN_USER_DATA = "MISSING_APPLICATION_VERSION_IN_USER_DATA";
 
     public static final String OUTDATED_TAUPAGE = "OUTDATED_TAUPAGE";
+
+    public static final String UNSECURED_ROOT_USER = "UNSECURED_ROOT_USER";
```
Comment (mrandi) [inline fullstop-apis/fullstop-violation-api/src/main/java/org/zalando/stups/fullstop/violation/ViolationType.java]: Reminder: create db migration script in order to get also a nice message

File: fullstop-jobs/src/main/java/org/zalando/stups/fullstop/jobs/iam/NoPasswordViolationWriter.java [hunk 2]
```diff
@@ -25,16 +29,39 @@ public NoPasswordViolationWriter(ViolationSink violationSink) {
...
+        metaInfo.put("arn", csvReportEntry.getArn());
+        metaInfo.put("is_password_enabled", String.valueOf(csvReportEntry.isPasswordEnabled()));
+        metaInfo.put("is_mfa_active", String.valueOf(csvReportEntry.isMfaActive()));
+        metaInfo.put("is_access_key_1_active", String.valueOf(csvReportEntry.isAccessKey1Active()));
+        metaInfo.put("is_access_key_2_active", String.valueOf(csvReportEntry.isAccessKey2Active()));
+
+        violationSink.put(
+                new ViolationBuilder()
+                        .withEventId("check-iam-user_" + csvReportEntry.getUser())
+                        .withAccountId("put here the account property")
```
Comment (mrandi) [inline fullstop-jobs/src/main/java/org/zalando/stups/fullstop/jobs/iam/NoPasswordViolationWriter.java]: here should be configured the property to get the account id that should than solve this violations.

File: fullstop-apis/fullstop-violation-api/src/main/java/org/zalando/stups/fullstop/violation/ViolationType.java (hunk 1 above)
Comment (mrandi) [inline fullstop-apis/fullstop-violation-api/src/main/java/org/zalando/stups/fullstop/violation/ViolationType.java]: :white_check_mark:

**Comment by Gregsen** [comment]
+1

**Comment by Gregsen** [comment]
:+1:

**Pull request merged**